### PR TITLE
add feature flags for Interactive Elements

### PR DESCRIPTION
**Pull request by nbogdanovskiydialog**

### CODE REVIEW

File: docs/feature-flags.schema.json [hunk 1]
```diff
@@ -410,6 +410,26 @@
       "title": "Schema of targeting form",
       "description": "JSON with description of UI schema for targeting form",
       "default": null
+    },
+      "chat.IE.MaxCount": {
```
Comment (olegshilov) [inline docs/feature-flags.schema.json]:
```suggestion
      "chat.interactiveElement.maxCount": {
```

File: docs/feature-flags.schema.json [hunk 2]
```diff
@@ -410,6 +410,26 @@
...
+      "type": "integer",
+      "title": "Maximum count of Interactive Elements in 1 message",
+      "default": 10
+    },
+      "chat.IE.MaxGroupCount": {
```
Comment (olegshilov) [inline docs/feature-flags.schema.json]:
```suggestion
      "chat. interactiveElement.maxGroupCount": {
```

File: docs/feature-flags.schema.json [hunk 3]
```diff
@@ -410,6 +410,26 @@
...
+      "type": "integer",
+      "title": "Maximum count of Group of Interactive Elements in 1 message",
+      "default": 10
+    },
+      "chat.IE.MaxCharCount": {
```
Comment (olegshilov) [inline docs/feature-flags.schema.json]:
```suggestion
      "chat.interactiveElement.maxCharCount": {
```

File: docs/feature-flags.schema.json [hunk 4]
```diff
@@ -410,6 +410,26 @@
...
+      "type": "integer",
+      "title": "Maximum count of characters in 1 Interactive Element",
+      "default": 32
+    },
+      "chat.IE.MaxCharGroupCount": {
```
Comment (olegshilov) [inline docs/feature-flags.schema.json]:
```suggestion
      "chat.interactiveElement.maxCharGroupCount": {
```

File: docs/feature-flags.schema.json (hunk 1 above)
Comment (pvlv) [inline docs/feature-flags.schema.json]: How about - `chat.media.interactiveMedia.` prefix ?

**Pull request merged**